sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/BigQueryIO.java

```diff
@@ -501,6 +502,21 @@ private static void dryRunQuery(BigQueryOptions options, String query) {
         return TableRowJsonCoder.of();
       }
 
+      @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
+        if (table != null) {
+          builder.add("tableSpec", toTableSpec(table));
+        }
+
+        if (query != null) {
+          builder.add("query", query);
+        }
+
+        if (flattenResults != null) {
+          builder.add("flattenResults", flattenResults);
+        }
```

Comment:
How do you want to handle options with default values? I would probably have done:

```java
if (!validation) {
   builder.add("validation", validation);
}
```
to only show this if validation is disabled.

(Or if there is a null-valued property where simply presence/absence of key is meaningful, just "validation disabled").